.gitignore

```diff
@@ -1,6 +1,14 @@
 .DS_Store
-.classpath
-.project
-.settings
-target
 test-output
+
+# Eclipse #
+/.classpath/
```

Comment:
The file `.classpath` is not a directory, and thus should not have a trailing slash.